ig/ig.go

```diff
@@ -0,0 +1,172 @@
+package ig
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+)
+
+type IG struct {
+	path  string
+	image string
+	v1    int
+	v2    int
+	v3    int
```

Comment:
You can use github.com/blang/semver to save and parse this information instead of keeping that as 3 integers and parsing by hand.